.pre-commit-config.yaml

```
@@ -38,7 +38,7 @@ repos:
         name: Check .bash files against bash-it requirements
         entry: ./hooks/dot-bash.sh
         language: system
-        files: "\\.ba[ts][sh]$"
+        files: "\\.(bash|bat)$"
```

```suggestion
        files: "\\.(bash|bats)$"
```

``.bats` should have meant the Bats test files. `*.bat` are Windows/MS-DOS batch files.